public/gameInfo.php

```diff
@@ -109,6 +109,7 @@
 }
 ?>
 <?php if ($gate): ?>
+    <?php RenderContentStart($pageTitle) ?>
```

Comment:
Probably not a big deal, but I noticed a discrepancy between the gate and the actual game page. 

Gate:
```
    <meta name="description" content="Adding achievements to retro games since 2012">
    <meta property="og:title" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
    <meta property="og:description" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
    <meta property="og:type" content="website">
 ```
 
 Page:
 ```
    <meta name="description" content="There are 54 achievements worth 275 points. ~Homebrew~ Larry and the Long Look for a Luscious Lover for NES - explore and compete on this classic game at RetroAchievements.">
    <meta property="og:title" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
    <meta property="og:description" content="There are 54 achievements worth 275 points. ~Homebrew~ Larry and the Long Look for a Luscious Lover for NES - explore and compete on this classic game at RetroAchievements.">
    <meta property="og:type" content="retroachievements:game">
 ```

This is consistent with the behavior before #1879.

Reply:
In latest, I've made `generateGameMetaDescription()` reusable between full pages and gated pages. Gated pages should now have the enhanced SEO meta description.